BugReporter/Serialization/SerializableDictionary.cs

```diff
@@ -69,7 +69,7 @@ public void WriteXml(XmlWriter writer)
         {
             foreach (var key in Keys)
             {
-                writer.WriteStartElement(key.ToString().Replace(" ", ""));
+                writer.WriteStartElement(key.ToString()!.Replace(" ", ""));
```

Comment:
📝 This appears to be a valid suppression due to a ["design choice"](https://github.com/dotnet/runtime/issues/1318) for `object.ToString()`